CONTRIBUTING.md

```diff
@@ -104,17 +104,16 @@ For pull request acceptance, you should:
 3.  Add a note to `CHANGELOG.md` about the changes. Include a link to the
     pull request.
 
-## Tips
+## Tox Tips
 
-1.  The primary tox environments for `watchmaker` include:
+1.  The _primary_ tox environments for `watchmaker` include:
 
     *   `check`
     *   `docs`
     *   `py26`
     *   `py27`
-    *   `py33`
-    *   `py34`
     *   `py35`
+    *   `py36`
 
```

Comment:
The spacing between the end of the bulleted list of 1) to 2) is not the same as the spacing in the rest of the list.